domains/alecks.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Alecks20",
+           "email": "109296824+Alecks20@users.noreply.github.com",
```

Comment:
Either remove your no-reply email or replace it with your own.

Reply:
Completed this